Cargo.toml

```diff
@@ -50,15 +50,16 @@ numpy = {version = "0.17", optional = true}
 indicatif = {version = "0.17", features = ["rayon"]}
 rstats = "1.2.50"
 thiserror = "1.0"
-parquet = "39.0.0"
+parquet = {version = "39.0.0", default-features = false, features = ["arrow"]}
 arrow = "39.0.0"
-shadow-rs = "0.21.0"
+shadow-rs = {version = "0.21.0", default-features = false}
 serde_yaml = "0.9.21"
 whoami = "1.3.0"
 either = {version = "1.8.1", features = ["serde"]}
 num = "0.4.0"
 enum-iterator = "1.4.0"
 rstest = "0.17.0"
+getrandom = {version = "0.2", features = ["js"]}
```

Comment:
What functionality does the `getrandom` crate add?

Reply:
It is used by the `rand` crate. If we do not specify this feature flag, it cannot be compiled for WASM.